.github/workflows/ruby.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [ 'develop' ]
+  pull_request:
+    branches: ['**']
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '3.3'
+          - '3.2'
```

Comment:
I think traction-service is on ruby 3.4 and rails is on 8.0 If its no extra work it would be good to add that but not essential. Maybe requires another story.

Reply:
We're using the `master` branch to create another gem for Traction (the second part of this story), so I thought it would be ok to align Traction specs with that branch specifically and stick to the Ruby and Rails versions that Sequencescape uses here. Hope that makes sense.